### PR TITLE
Hw27

### PR DESCRIPTION
**Pull request by Engelko**

### CODE REVIEW

**Review by Engelko — COMMENTED** [review]
I accidentally made a commit for hw23 on the wrong branch

File: Bash_scripts/HW27/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+        stage ("docker build image") {
+            steps {
+                echo "=========docker build image========"
+                dir ('Bash_scripts/HW27') {
+                    sh 'docker build -t hw27testing .'
+                }
+            }
+        }
+    }
+}
```
Comment (mavrik90) [inline Bash_scripts/HW27/Jenkinsfile]: new line

Comment (Engelko) [reply Bash_scripts/HW27/Jenkinsfile]: fixed

**Review by mavrik90 — APPROVED** [review]
Nice!

**Pull request merged**